src/core/evaluator.js

```diff
@@ -422,8 +422,16 @@ class PartialEvaluator {
         smask,
         isolated: false,
         knockout: false,
+        optionalContent: null,
       };
 
+      if (dict.has("OC")) {
+        groupOptions.optionalContent = this.parseMarkedContentProps(
```

Comment:
Nit: With parseMarkedContentProps being asynchronous, as suggested below, this would need to be `groupOptions.optionalContent = await this.parseMarkedContentProps(` instead.